server/core/db/boltdb.go

```diff
@@ -75,18 +81,20 @@ func NewWithDB(db *bolt.DB, bucket string, globalBucket string) (*BoltDB, error)
 	return &BoltDB{
 		db:                    db,
 		locksBucketName:       []byte(bucket),
+		queueBucketName:       []byte(queueBucketName),
 		pullsBucketName:       []byte(pullsBucketName),
 		globalLocksBucketName: []byte(globalBucket),
 	}, nil
 }
 
 // TryLock attempts to create a new lock. If the lock is
-// acquired, it will return true and the lock returned will be newLock.
-// If the lock is not acquired, it will return false and the current
-// lock that is preventing this lock from being acquired.
-func (b *BoltDB) TryLock(newLock models.ProjectLock) (bool, models.ProjectLock, error) {
+// acquired, it will return true, the lock returned which will be newLock.
```

Comment:
@ghaiszaher minor, but this line reads weird, was this meant?

```suggestion
// acquired, it will return true and the newLock.
```

Reply:
Thanks, I'll update in https://github.com/runatlantis/atlantis/pull/2782